services/web/server/src/simcore_service_webserver/projects/_db_utils.py

```diff
@@ -257,13 +177,12 @@ async def _upsert_tags_in_project(
                 .on_conflict_do_nothing()
```

Comment:
Question: hmm in that case. and if I got it right you removed almost all the permission checks. why not just remove that stuff? or not talk about it?

Reply:
actually are you sure it is still used?

Reply:
Yes it is used and I just removed the one line regarding permission check, I would not like to touch it more.